src/electron/index.ts

```diff
@@ -414,15 +414,16 @@ function main() {
   // Disconnects from the current server, if any.
   promiseIpc.on('stop-proxying', stopVpn);
 
-  // Error reporting.
-  // This config makes console (log/info/warn/error - no debug!) output go to breadcrumbs.
-  ipcMain.on('environment-info', (event: Event, info: {appVersion: string, dsn: string}) => {
-    if (info.dsn) {
-      sentry.init({dsn: info.dsn, release: info.appVersion, maxBreadcrumbs: 100});
-    }
-    // To clearly identify app restarts in Sentry.
-    console.info(`Outline is starting`);
-  });
+// Error reporting.
+// This config makes console (log/info/warn/error - no debug!) output go to breadcrumbs.
+// Error reporting to Senty is disabled in debug mode
+ipcMain.on('environment-info', (event: Event, info: {appVersion: string, dsn: string}) => {
+  if (info.dsn && !debugMode) {
```

Comment:
Can we instead change the `console.[log|error]` statements in the process handlers to `console.debug`? That way we get non-process logs and crash reports.

Also, nit: indentation.

Reply:
This makes it hard to redirect the logs to a file.  Instead I went back to the previous implementation from the last PR to directly pipe the subprocess stdout and stderr to the electron stdout and stderr, bypassing Sentry.  This still allows collecting the debug logs in a file with 

```cmd
set OUTLINE_DEBUG=true
“c:\Program Files (x86)\Outline\Outline.exe” 1>%TEMP%\outline.log 2>&1
```

in `cmd`